Makefile

```diff
@@ -0,0 +1,38 @@
+APP_ID := org.speedcrunch.SpeedCrunch
+RUNTIMES := org.kde.Sdk//5.11 org.kde.Platform//5.11
+
+BUILD := build
+DIST := dist
+REPO := $(BUILD)/repo
+
+all: $(REPO) $(DIST)/$(APP_ID).flatpak $(DIST)/$(APP_ID).Debug.flatpak
+
+# ----
+
+install: $(BUILD)/$(APP_ID).flatpak
+	flatpak install -y --user $(BUILD)/$(APP_ID).flatpak
+
+$(REPO): *.yaml
+	flatpak-builder --force-clean --ccache --repo=$@ $(BUILD)/build --state-dir=$(BUILD)/.flatpak-builder $(APP_ID).yaml
+	flatpak build-update-repo $(REPO)
+
+$(BUILD)/$(APP_ID).flatpak: $(REPO)
+	flatpak build-bundle $(REPO) $@ $(APP_ID)
+
+$(BUILD)/$(APP_ID).Locale.flatpak: $(REPO)
+	flatpak build-bundle $(REPO) $@ $(APP_ID).Locale --runtime
+
+$(BUILD)/$(APP_ID).Debug.flatpak: $(REPO)
+	flatpak build-bundle $(REPO) $@ $(APP_ID).Debug --runtime
+
+$(DIST):
+	mkdir -p $(DIST)
+
+$(DIST)/%.flatpak: $(BUILD)/%.flatpak $(DIST)
+	cp $< $@
+
+setup:
+	flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+	flatpak install -y flathub $(RUNTIMES)
+
+.PHONY: all install setup
```

Comment:
It's in the repo because it's useful for local building. Flathub doesn't care, but if there's one thing I hate, it's remembering things. And if there's one thing I hate more than remembering things, it's remembering build commands. ;)

FWIW: my previous app submission had effectively the same Makefile in the repo.

Reply:
> It's in the repo because it's useful for local building. Flathub doesn't care, but if there's one thing I hate, it's remembering things. And if there's one thing I hate more than remembering things, it's remembering build commands. ;)

I totally understand you because I have the same problem. Unfortunately, it seems that this is not allowed here anymore. My pull requests were blocked for months only because of containing helper scripts and/or README describing the building process. In the end, I decided to remove these files, seeing that it doesn't lead to anything.

Official explanation:
> It unnecessarily complicates things that are accomplished with one command.

> Clarity of what's going on is important, especially on Flathub, which we want to be seen as source of good practices.

For this reason, `shell wrappers over common commands` (AKA `shell wrappers that are not simplifying anything` or `bash helpers that shouldn't be part of manifest`) and `readme that duplicates official documentation` (AKA `unneeded readme that repeats documentation`) are not allowed here.

> FWIW: my previous app submission had effectively the same Makefile in the repo.

As @barthalion said:
> Rules are fluid and just because something has been accepted before doesn't mean it would be now.

> As I said previously, rules aren't set in stone and not everything would be accepted as is today.

Therefore, I suggest adding this `Makefile` to the `.gitignore`, because otherwise you may get stuck with this pull request for months.